protocol/contracts/ecosystem/ShipmentPlanner.sol

```diff
@@ -0,0 +1,69 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.20;
+
+/**
+ * @notice Constraints of how many Beans to send to a given route at the current time.
+ * @param points Weight of this shipment route relative to all routes. Expects precision of 1e18.
+ * @param cap Maximum Beans that can be received by this stream at this time.
+ */
+struct ShipmentPlan {
+    uint256 points;
+    uint256 cap;
+}
+
+interface IBeanstalk {
+    function isFertilizing() external view returns (bool);
+    function totalUnfertilizedBeans() external view returns (uint256);
+
+    function isHarvesting(uint256 fieldId) external view returns (bool);
+    function totalUnharvestable(uint256 fieldId) external view returns (uint256);
+}
+
+uint256  constant BARN_POINTS = 333_333_333_333_333_333;
```

Comment:
1) remove extra space
2) should the constants lie inside `ShipmentPlanner`?

Reply:
fixed spacing and moved inside contract

https://github.com/BeanstalkFarms/Beanstalk/pull/860/commits/d65b6bacc7e6d96105a48c759af2f5222b0ccb26#diff-cf3f4898f41eee71e7b37cb1f5e642c532b4efd2ce1ee23de550f167e61ea6b5R32-R34